.storybook/preview-head.html

```diff
@@ -1,5 +1,8 @@
 <link rel="stylesheet" href="https://eu-central-1.linodeobjects.com/atlas-assets/fonts/PxGrotesk/PxGrotesk.css" />
 <link rel="stylesheet" href="https://eu-central-1.linodeobjects.com/atlas-assets/fonts/Inter/inter.css" />
+<script>
+  window.setImmediate = window.setTimeout
+</script>
```

Comment:
Basically, when you run `build-storybook` this script tag will be added to `iframe.html` inside `storybook-static`

Reply:
Not sure if you're aware but we actually do the same thing in our main `index.html`:
```
    <!--  manual polyfills  -->
    <script>
      window.setImmediate = setTimeout
      window.global = window
      window.__DEV__ = false
    </script>
    ```